### ISSUE TITLE
missing activation function?

### ISSUE THREAD

**Issue by jiangxiluning** [jiangqy/LSTM-Classification-Pytorch#1]
In the Following code, you output Linear layer's neuron's output directly. I saw somebody added log-softmax for multi-classes  classification and even I simply added a sigmod function after Linear layer with only one neuron. Could you explain your thoughts, thanks?

https://github.com/jiangqy/LSTM-Classification-Pytorch/blob/d4e5440f78fc0a3317c1137d070be1772fd01f16/utils/LSTMClassifier.py#L33

```python
def forward(self, sentence):
        embeds = self.word_embeddings(sentence)
        x = embeds.view(len(sentence), self.batch_size, -1)
        lstm_out, self.hidden = self.lstm(x, self.hidden)
        y  = self.hidden2label(lstm_out[-1])   # without activation function?
        return y
```

**Comment by jiangqy** [comment]
@jiangxiluning I use [CrossEntropyLoss](http://pytorch.org/docs/master/nn.html) which is combination of LogSoftMax and NLLLoss for pytorch implementation.

**Comment by jiangxiluning** [comment]
Thanks. But I think adding a logsoftmax layer after linear layer could be more clear for the multi-classes classification problem.
